mis_builder_cash_flow/data/mis_report.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+  <record id="mis_report_cash_flow" model="mis.report">
+    <field name="name">Cash Flow</field>
+    <field name="style_id" ref="mis_style_cash_flow"/>
+  </record>
+
+  <record id="mis_kpi_acum_accounting" model="mis.report.kpi">
+    <field name="report_id" ref="mis_report_cash_flow"/>
+    <field name="name">acum_accounting</field>
+    <field name="description">Acum Accountiong</field>
```

Comment:
I use accumulated instead of total to differentiate between the totals of the period and the accumulated balance from always. This is an auxiliary line, no problem to change it to total, I just believe better accum (yes I think I should use Accumulated instead of "acum")

Reply:
@jjscarafia I understand (I definitively wanted to get rid of the "acum"). Not sure Accumulated is the best in English. "Running Total" or " Cumulative Total " might be better but I am not sure so up to you :smile:

Reply:
Done, I've remove this lines...